demo/src/app/components/carousel/demos/config/carousel-config.ts

```diff
@@ -7,6 +7,8 @@ import {NgbCarouselConfig} from '@ng-bootstrap/ng-bootstrap';
   providers: [NgbCarouselConfig] // add NgbCarouselConfig to the component providers
 })
 export class NgbdCarouselConfig {
+  images = [1, 2, 3].map(() => `https://picsum.photos/900/500?image=${Math.floor(Math.random() * 1000)}`);
```

Comment:
The problem with random is that sometimes the images are not there. I guess it's more of an exception, but this one broke for me: https://picsum.photos/900/500?image=207

<img width="400" alt="screen shot 2018-03-09 at 09 42 24" src="https://user-images.githubusercontent.com/8074436/37198360-59a365bc-237e-11e8-89e8-260e318e0e19.png">

I still like random images though...